tools/update-generated-files.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * Script that finds all `generated_file_test` targets and runs their `.update` targets.
+ * This is useful if a dependency that is bundled into all checked-in bundles has been
+ * updated. All of the checked-in bundles would need to be updated and it's cumbersome
+ * to manually run all of these targets.
+ */
+
+import {join} from 'path';
+import {spawnSync, SpawnSyncOptionsWithStringEncoding} from 'child_process';
+
+const projectDir = join(__dirname, '../');
+const bazelPath = process.env.BAZEL ?? join(projectDir, 'node_modules/.bin/bazel');
```

Comment:
Note: I considered using `yarn bazel` here, or just calling `getBazelBinary` from `@bazel/bazelisk`, but that will complicate the setup with `spawnSync`. `execSync` is not windows-compatible with `bazel query` (this is the reason the golden manage scripts in FW and CLI do not work for Windows)